### PR TITLE
Luxury bar cruiser

### PR DESCRIPTION
**Pull request by rhodochrositech**
## About The Pull Request
This pull request adds a pre-fixed up whiteship with a dark secret. Designed as an admin-event ship providing a high roleplay environment including a bar bouncer role and captain role, this ship can provide quite an adventure and roleplay experience.

Adds The Barrows, a whiteship with a bar and lounge designed for luxury.
Equipped with:
Identification Cards, Uniforms
Self Defense Weaponry
Fully stocked medical bay
A beautiful 5 star bar
A fine dining experience with a view of open space
A robust security system
Employment Opportunities with wonderful medical benefits
A couple of interesting easter eggs

## Why It's Good For The Game
Adds content, environment, and a great idea for low population controlled antaggery that guides proper escalation (Debt type) while constructing it around the concept of a job.
barrows

## Changelog
:cl:
add: Adds a Whiteship, The Barrows (V1), a mobile ship ready to provide a roleplay medical mystery at the bar.
/:cl:

### CODE REVIEW

File: code/datums/shuttles.dm [hunk 1]
```diff
@@ -708,3 +708,7 @@
 	suffix = "crow"
 	name = "ESS Crow"
 	who_can_purchase = null
+/datum/map_template/shuttle/barrow
+	port_id = "whiteship"
+	suffix = "the_barrows"
+	name = "The Barrows"
```
Comment (atakiya) [inline code/datums/shuttles.dm:714]:
```suggestion

/datum/map_template/shuttle/barrow
	port_id = "whiteship"
	suffix = "the_barrows"
	name = "The Barrows"

```
newlines around blocks please.

File: code/game/area/areas/shuttles.dm [hunk 2]
```diff
@@ -289,3 +289,14 @@
...
+/area/shuttle/barrow
+	name = "The Barrows"
+	requires_power = FALSE
+	area_limited_icon_smoothing = /area/shuttle/barrow
+/area/shuttle/barrow/helm
+	name = "The Barrows Helm"
+/area/shuttle/barrow/medbay
+	name = "The Barrows Medbay"
+/area/shuttle/barrow/bar
+	name = "The Barrows Bar"
```
Comment (atakiya) [inline code/game/area/areas/shuttles.dm:302]:
```suggestion

/area/shuttle/barrow
	name = "The Barrows"
	requires_power = FALSE
	area_limited_icon_smoothing = /area/shuttle/barrow

/area/shuttle/barrow/helm
	name = "The Barrows Helm"

/area/shuttle/barrow/medbay
	name = "The Barrows Medbay"

/area/shuttle/barrow/bar
	name = "The Barrows Bar"
```
As well.

File: code/modules/shuttle/white_ship.dm [hunk 3]
```diff
@@ -64,3 +64,14 @@
...
+/obj/machinery/computer/shuttle/barrow
+	name = "The Barrows Ship Console"
+	desc = "Used to control The Barrows."
+	circuit = /obj/item/circuitboard/computer/barrow
+	shuttleId = "whiteship_the_barrows"
+	
+/obj/item/circuitboard/computer/barrow
+	name = "The Barrows Ship Console (Computer Board)"
+	greyscale_colors = CIRCUIT_COLOR_GENERIC
+	build_path = /obj/machinery/computer/shuttle/barrow
```
Comment (atakiya) [inline code/modules/shuttle/white_ship.dm:77]:
```suggestion
	build_path = /obj/machinery/computer/shuttle/barrow

```
EOL on newline.

**Review by atakiya — CHANGES_REQUESTED** [review]
A few codestyle issues only.

---
If you use vscode or anoher editor with plugin support, you can install an extension/plugin that can read `.editorconfig` files to automatically fix code formatting without manual intervention.

File: code/modules/shuttle/white_ship.dm (hunk 3 above)
Comment (rhodochrositech) [reply code/modules/shuttle/white_ship.dm:77]: Thank you - resolving now. Sending in for review in a moment!

**Review by atakiya — APPROVED** [review]
Changes applied.
Changelog fixed.
CI passed.

Good to go 🚀

**Pull request merged**